common/utils.py

```diff
@@ -1,29 +1,53 @@
-import aiohttp
-import discord
+from datetime import datetime
 import json
 import os
+import aiohttp
+import discord
+from dotenv import load_dotenv
+import pytz
 
 # Frequently used Objects
 intents = discord.Intents.default()
 intents.members = True
 intents.message_content = True
 client = discord.Client(intents=intents)
-env = None
+commandTree = discord.app_commands.CommandTree(client)
+load_dotenv()
+env = {
+    "TOKEN": os.getenv("DISCORD_TOKEN"),
+    "GUILD": os.getenv("DISCORD_GUILD"),
+    "BOT_ID": os.getenv("BOT_ID"),
+    "ADMIN_ROLE": os.getenv("ADMIN_ROLE"),
+    "MAIN_CHANNEL": os.getenv("MAIN_CHANNEL"),
+    "BOT_CHANNEL": os.getenv("BOT_CHANNEL"),
+    "ANNOUNCEMENT_CHANNEL": os.getenv("ANNOUNCEMENT_CHANNEL"),
+    "ANNOUNCEMENT_DAY": int(os.getenv("ANNOUNCEMENT_DAY")),
+    "ANNOUNCEMENT_HOUR": int(os.getenv("ANNOUNCEMENT_HOUR")),
+    "ANNOUNCEMENT_MIN": int(os.getenv("ANNOUNCEMENT_MIN")),
+    "WELCOME_ROLE": os.getenv("WELCOME_ROLE"),
+    "DOTA_CHANNEL": os.getenv("DOTA_CHANNEL"),
+    "TWITCH_CLIENT_ID": os.getenv("TWITCH_CLIENT_ID"),
+    "TWITCH_CLIENT_SECRET": os.getenv("TWITCH_CLIENT_SECRET"),
+    "MEME_CHANNEL": os.getenv("MEME_CHANNEL"),
+    "YOUTUBE_API_KEY": os.getenv("YOUTUBE_API_KEY"),
+    "MOVIE_CHANNEL": os.getenv("MOVIE_CHANNEL"),
+    "MOVIE_ROLE": os.getenv("MOVIE_ROLE"),
+}
 botObject = None
 guildObject = None
 mainChannel = None
 botChannel = None
+# Color for Embedded Messages
+embed_colour = {"MOVIE_NIGHT": 0x4F4279, "ERROR": 0xED4337}
+MOVIE_EVENT_NAME = "Movie Night"
+DELETE_AFTER_SECONDS = 10 * 60
+DELETE_AFTER_HOURS = 24 * 60 * 60
```

Comment:
Let's leave better names here so that it's easier to resuse. `EXTENDED_MESSAGE_DURATION`?

Reply:
Done